apps/beeswax/src/beeswax/server/hive_server2_lib_tests.py

```diff
@@ -298,9 +298,11 @@ def test_get_table_with_error(self):
         client.get_table(database='database', table_name='table_name')
       except QueryServerException as e:
         if sys.version_info[0] > 2:
-          req_string = "TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'1', secret=b'1')), catalogName=None, schemaName='database', tableName='table_name', tableTypes=None)"
+          req_string = ("TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'l\\xc4', secret=b'l\\xc4')), "
+            "catalogName=None, schemaName='database', tableName='table_name', tableTypes=None)")
         else:
-          req_string = "TGetTablesReq(schemaName='database', sessionHandle=TSessionHandle(sessionId=THandleIdentifier(secret='1', guid='1')), tableName='table_name', tableTypes=None, catalogName=None)"
+          req_string = ("TGetTablesReq(schemaName='database', sessionHandle=TSessionHandle(sessionId=THandleIdentifier"
+            "(secret='1', guid='1')), tableName='table_name', tableTypes=None, catalogName=None)")
         assert_equal(
           "Bad status for request %s:\n%s" % (req_string, get_tables_res),
           str(e)
```

Comment:
Here what happens if just looking at e.message?

Reply:
this is the output of e.message `Bad status for request TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'l\xc4', secret=b'l\xc4')), catalogName=None, schemaName='database', tableName='table_name', tableTypes=None):`